src/ORM/Locator/TableLocator.php

```diff
@@ -160,8 +160,14 @@ public function get(string $alias, array $options = []): Table
      */
     protected function createInstance(string $alias, array $options)
     {
-        [, $classAlias] = pluginSplit($alias);
-        $options = ['alias' => $classAlias] + $options;
+        if (strpos($alias, '\\') === false) {
+            [, $classAlias] = pluginSplit($alias);
+            $options = ['alias' => $classAlias] + $options;
+        } elseif (!isset($options['alias'])) {
+            $options['className'] = $alias;
+            /** @psalm-suppress PossiblyFalseOperand */
+            $alias = substr($alias, strrpos($alias, '\\') + 1, -5);
```

Comment:
What does this look like when you give it an invalid table class? Can we add a test for that?

Reply:
Looks like the last 5 characters will get chopped off.

Reply:
It wouldnt be worse than the current code, for sure.
But we can also check and throw a meaningful exception in that case.

Reply:
I copied this code from `ModelAwareTrait::loadModel()`. 

For an invalid class name the method will return a `Table` instance with whatever alias is generated after this string manipulation. I don't think any extra defensive coding is necessary here.

Reply:
Sounds good. I don't want extra coding either ... was just curious what error the user would see.